.github/workflows/production.yml

```diff
@@ -52,7 +52,28 @@ jobs:
               # For using same tag for staging and production we need to uncomment these two below lines:
               # with:
               #   tagRegex: "production(.*)"
+            - name: Building docker image 🐳
+              run: docker build -t ${{ secrets.DOCKHUB_ORGANISATION }}/deriv-com:latest -t ${{ secrets.DOCKHUB_ORGANISATION }}/deriv-com:$GIT_TAG_NAME .
 
+            - name: Verify nginx image
+              run: |
+                  set -e
+                  docker run --rm ${{ secrets.DOCKHUB_ORGANISATION }}/deriv-com:$GIT_TAG_NAME nginx -t
+                  echo "docker image validated successfully"
+
+            - name: Pushing Image to docker hub 🐳
+              run: |
+                  echo ${{ secrets.DOCKERHUB_PASSWORD }}| docker login -u ${{ secrets.DOCKERHUB_USERNAME }} --password-stdin
+                  docker push  ${{ secrets.DOCKHUB_ORGANISATION }}/deriv-com:latest
+                  docker push  ${{ secrets.DOCKHUB_ORGANISATION }}/deriv-com:$GIT_TAG_NAME
+
+            - name: Deploy 🚀
+              run: |
+                  export NAMESPACE="deriv-com-production"
+                  git clone https://github.com/binary-com/devops-ci-scripts
+                  cd devops-ci-scripts/k8s-build_tools
+                  echo ${{ secrets.CA_CRT}} | base64 --decode > ca.crt
+                  ./release.sh deriv-com $GIT_TAG_NAME
```

Comment:
@yaswanth-deriv please move them to after publish to cloudflare.
the reason is that we dont want to block the releases at least for now until it gets to a stable point.